### PR TITLE
Added "tsh status" command.

### PR DESCRIPTION
**Pull request by russjones**
**Purpose**

Show the proxy the user is logged into and metadata from the certificate.

**Implementation**

The output of `tsh status` after logging in looks like this:

```
$ tsh status
localhost
----------
User:       rjones
Roles:      admin
Logins:     rjones, root
Expires:    11h40m0s
Extensions: permit-agent-forwarding, permit-port-forwarding, permit-pty
Format:     standard
```

**Related Issues**

Fixes https://github.com/gravitational/teleport/issues/1628

**EDIT [ev]** see my comment at the bottom of the discussion

### CODE REVIEW

**Comment by klizhentas** [comment]
your ouptut is missing tabbing

File: tool/tsh/tsh.go [hunk 1]
```diff
@@ -225,6 +225,10 @@ func Run(args []string, underTest bool) {
 	show := app.Command("show", "Read an identity from file and print to stdout").Hidden()
 	show.Arg("identity_file", "The file containing a public key or a certificate").Required().StringVar(&cf.IdentityFileIn)
 
+	// The status command shows which proxy the user is logged into and metadata
+	// about the certificate.
+	status := app.Command("status", "Show which proxy and certificates for user.")
```
Comment (klizhentas) [inline tool/tsh/tsh.go]: Show which proxy -> should be something else

File: tool/tsh/tsh.go [hunk 2]
```diff
@@ -819,3 +825,73 @@ func onShow(cf *CLIConf) {
 
 	fmt.Printf("Fingerprint: %s\n", ssh.FingerprintSHA256(pub))
 }
+
+// onStatus command shows which proxy the user is logged into and metadata
+// about the certificate.
+func onStatus(cf *CLIConf) {
+	tc, err := makeClient(cf, true)
+	if err != nil {
+		utils.FatalError(err)
```
Comment (klizhentas) [inline tool/tsh/tsh.go]: I'm not sure about using FatalError everywhere , for example your defer won't be called and you will have a connection hanging. Can you wrap this into fucnction that returns proper error, and then fatal exit on the other layer.

File: tool/tsh/tsh.go [hunk 3]
```diff
@@ -819,3 +825,73 @@ func onShow(cf *CLIConf) {
...
+	}
+
+	proxyClient, err := tc.ConnectToProxy(cf.Context)
```
Comment (klizhentas) [inline tool/tsh/tsh.go]: why do you need to connect to the remote cluster for this command? I was thinking to local profile information for this purpose.

**Review by klizhentas — CHANGES_REQUESTED** [review]
have some questions/comments

File: tool/tsh/tsh.go (hunk 3 above)
Comment (klizhentas) [reply tool/tsh/tsh.go]: Specifically, if client is logged into multiple proxies, we want to show this as well

Comment (russjones) [reply tool/tsh/tsh.go]: The users roles are not always available without connecting to the proxy. They are only available in the certificate for new style certificates.

Comment (klizhentas) [reply tool/tsh/tsh.go]: then don't show them for compat mode

File: tool/tsh/tsh.go (hunk 2 above)
Comment (klizhentas) [reply tool/tsh/tsh.go]: not the connection hanging, but connection state hanging

Comment (klizhentas) [reply tool/tsh/tsh.go]: it will be eventually GCed but still

**Comment by kontsevoy** [comment]
Slightly improved output:

```
> Proxy URL      https://proxy.example.com:3080
  Logged in as:  ev@kontsevoy.com
  Roles:         admin*
  Logins:        rjones, root
  Valid until:   03/04/2018 2:45pm PST [EXPIRED]
  Extensions:    permit-agent-forwarding, permit-port-forwarding, permit-pty

  Proxy URL      https://another.example.com:3080
  Logged in as:  ev@kontsevoy.com
  Roles:         admin*
  Logins:        rjones, root
  Valid until:   03/04/2018 2:45pm PST [EXPIRED]
  Extensions:    permit-agent-forwarding, permit-port-forwarding, permit-pty


* RBAC is only available in Teleport Enterprise
  https://gravitaitonal.com/teleport/docs/enteprise
```

**Review by kontsevoy — APPROVED** [review]

**Comment by russjones** [comment]
I'm going to close this, update the ticket with the new format. Then I'll create a new PR Monday.

**Pull request closed without merging**